package.json

```diff
@@ -3,43 +3,44 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@types/node": "^16.11.11",
-    "@types/react": "^17.0.37",
+    "@types/node": "^16.11.19",
+    "@types/react": "^17.0.38",
     "@types/react-dom": "^17.0.11",
     "classnames": "^2.3.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "sass": "^1.44.0",
+    "react-scripts": "5.0.0",
+    "sass": "^1.48.0",
     "serve": "^13.0.2",
-    "typescript": "^4.5.4"
+    "typescript": "^4.5.4",
+    "web-vitals": "^2.1.3"
   },
   "devDependencies": {
     "@testing-library/dom": "^8.11.1",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.0.3",
-    "eslint": "^7.32.0",
+    "@types/jest": "^27.4.0",
+    "eslint": "^8.6.0",
     "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-jest": "^25.3.0",
+    "eslint-plugin-jest": "^25.3.4",
     "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-testing-library": "^5.0.1",
+    "eslint-plugin-testing-library": "^5.0.3",
     "husky": "^7.0.4",
     "jest-junit": "^13.0.0",
     "lint-staged": "^12.1.2",
     "npm-run-all": "^4.1.3",
     "prettier": "2.5.1",
     "shelljs": "^0.8.5",
-    "stylelint": "^13.12.0",
-    "stylelint-config-recommended-scss": "^4.3.0",
-    "stylelint-config-standard": "^22.0.0",
+    "stylelint": "^14.2.0",
+    "stylelint-config-recommended-scss": "^5.0.2",
+    "stylelint-config-standard": "^24.0.0",
     "stylelint-declaration-strict-value": "^1.8.0",
-    "stylelint-scss": "^3.21.0"
+    "stylelint-scss": "^4.1.0"
   },
   "resolutions": {
-    "@typescript-eslint/typescript-estree": "^5.6.0",
```

Comment:
I dropped this resolution `@typescript-eslint/typescript-estree` because I wasn't sure why it was in there, and getting rid of it didn't seem to have an effect.